scripts/build.sh

```diff
@@ -36,11 +36,10 @@ if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]
   set +e
   git add .
   set -e
-  git status
```

Comment:
@ryanseys : Is this required? I removed so that I could comment on it in this PR.